packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx

```diff
@@ -28,67 +28,67 @@ export const WebhookToggleField: React.FC<FieldProps> = ({ register, watch, setV
   </StackedList.Root>
 )
 
-export const WebhookNameField: React.FC<FieldProps & { disabled: boolean }> = ({ register, errors, disabled }) => (
+export const WebhookNameField: React.FC<WebhookFormFieldProps & { disabled: boolean }> = ({
```

Comment:
any thoughts on if the form framework can render most this form without requiring custom components?

Reply:
Can be done, but it would be one huge file with 500 lines of code (harder to manage errors, any speicifc changes etc), so i broke it up and added them as separate components called "fields" . All of these form components exist in the same file.  

imho it's easier to debug and maintain when each field exists in a silo of its own.